CMakeLists.txt

```diff
@@ -103,6 +103,18 @@ if (NOT ONEDPL_BACKEND)
     message(STATUS "Using ${ONEDPL_BACKEND} as default backend")
 endif()
 
+if (NOT ONEDPL_ENABLE_PSTL_OFFLOAD)
```

Comment:
This option will need to be added to `cmake/README.md` to explain its usage.   
Also, to be clear, the `find_package(oneDPL)` method of cmake inclusion ***does not*** use this CMake file as a baseline.  Any visibility / availability of this feature to that method of inclusion will need to be added independently to `cmake/templates/oneDPLConfig.cmake.in`.  

That method of inclusion uses the `dpcpp` backend whenever SYCL is found to be available,  `ONEDPL_BACKEND` is *not* an option provided by the user.  Instead, `ONEDPL_PAR_BACKEND` is selected which chooses the standard host (par) backend.  

I think you could support the same option, and only attempt to turn it on when `_sycl_support` is true, ***or*** you could provide pstl_offload as another possible option of `ONEDPL_PAR_BACKEND`.   Either way will require adding some code to `cmake/templates/oneDPLConfig.cmake.in`.

Reply:
We have discussed this with Ruslan and it seems like we don't want to publish these options. 
The only available option for user to get pstl offload working is to have the correct compiler that supports `-fsycl-pstl-offload` option and oneDPL package with official packaging,  vars and layout.
Taking oneDPL from source and use together with `-fsycl-pstl-offload` is not supposed to be supported (at least for today).
So in that case the only thing why I have added CMake part is to simplify building the binary and testing on our side because we are using oneDPL testing infrastructure to test pstl-offload.
From the user perspective, these options are considered private.

Reply:
Aha, I understand.  So really for the other method of inclusion (`find_package`), and after the compiler option `-fsycl-pstl-offload` is available, it is the user's responsibility to just add that option and it should just work.  No changes are required for oneDPL cmake from that direction.  Is this correct?

Also, with this in mind, do we want to "uglify" the option in some fashion to denote that it is "private"?  I don't have a strong opinion to do this, I'm just asking the question.

Reply:
Yes, that is correct regarding the option. The user should manually add `-fsycl-pstl-offload` for the code that is intended to use pstl offloading.
Regarding the uglification, I think is sounds reasonable. I will check if we have other private options and try to use the same approach (or invent my own one if no such options exists:) )

Reply:
Added option uglification